### PR TITLE
Make recently added help strings dynamic, add one missing string

### PR DESCRIPTION
**Pull request by UdjinM6**
A followup for #3981

### CODE REVIEW

**Review by xdustinface — APPROVED** [review]
utACK

nit: The current title doesn't make 671f6ce2b52655af41806778a0f95f68d7c8abf6 looking related to this PR :)

**Comment by UdjinM6** [comment]
> nit: The current title doesn't make [671f6ce](https://github.com/dashpay/dash/commit/671f6ce2b52655af41806778a0f95f68d7c8abf6) looking related to this PR :)

Hmmm... true...

**Comment by UdjinM6** [comment]
Moved it to #3997

**Review by xdustinface — APPROVED** [review]
utACK

File: src/init.cpp [hunk 1]
```diff
@@ -600,12 +610,14 @@ std::string HelpMessage(HelpMessageMode mode)
...
+        strUsage += HelpMessageOpt("-highsubsidyfactor=<n>", strprintf(_("The factor to multiply the normal block subsidy by while in the highsubsidyblocks window of a devnet (default: %u)"), devnetConsensus.nHighSubsidyFactor));
+        strUsage += HelpMessageOpt("-llmqchainlocks=<quorum name>", strprintf(_("Override the default LLMQ type used for ChainLocks on a devnet. Allows using ChainLocks with smaller LLMQs. (default: %s)"), devnetConsensus.llmqs.at(devnetConsensus.llmqTypeChainLocks).name));
+        strUsage += HelpMessageOpt("-llmqdevnetparams=<size:threshold>", strprintf(_("Override the default LLMQ size for the LLMQ_DEVNET quorum (default: %u:%u)"), devnetLLMQ.size, devnetLLMQ.threshold));
+        strUsage += HelpMessageOpt("-llmqinstantsend=<quorum name>", strprintf(_("Override the default LLMQ type used for InstantSend on a devnet. Allows using InstantSend with smaller LLMQs. (default: %s)"), devnetConsensus.llmqs.at(devnetConsensus.llmqTypeInstantSend).name));
+    }
+    strUsage += HelpMessageOpt("-llmqtestparams=<size:threshold>", strprintf(_("Override the default LLMQ size for the LLMQ_TEST quorum (default: %u:%u)"), regtestLLMQ.size, regtestLLMQ.threshold));
```
Comment (PastaPastaPasta) [inline src/init.cpp:620]: I'm not sure I like the idea of running `-devnet="xyz" -help` just to see the help text..

File: src/init.cpp [hunk 2]
```diff
@@ -458,6 +458,16 @@ std::string HelpMessage(HelpMessageMode mode)
...
     const auto testnetChainParams = CreateChainParams(CBaseChainParams::TESTNET);
+
+    Consensus::Params devnetConsensus;
+    Consensus::LLMQParams devnetLLMQ;
+    if (gArgs.IsArgSet("-devnet")) {
+        devnetConsensus = CreateChainParams(CBaseChainParams::DEVNET)->GetConsensus();
+        devnetLLMQ = devnetConsensus.llmqs.at(Consensus::LLMQ_DEVNET);
+    }
+
+    const auto& regtestLLMQ = CreateChainParams(CBaseChainParams::REGTEST)->GetConsensus().llmqs.at(Consensus::LLMQ_TEST);
```
Comment (PastaPastaPasta) [inline src/init.cpp:469]: Maybe I'm missing something, but I don't see how this is relevant

Comment (UdjinM6) [reply src/init.cpp:469]: It's used for `-llmqtestparams` help sptring

File: src/init.cpp (hunk 1 above)
Comment (UdjinM6) [reply src/init.cpp:620]: Could use `-devnet -?` as a shorter version but yes, it's unfortunate. The reason for wrapping it with `if (gArgs.IsArgSet("-devnet"))` is that creating devnet params triggers `FindDevNetGenesisBlock` which calls `GetDevNetName()` which requires `-devnet` to be set (`assert(IsArgSet("-devnet"));`).

**Review by thephez — DISMISSED** [review]
utACK

File: src/init.cpp (hunk 1 above)
Comment (PastaPastaPasta) [reply src/init.cpp:620]: Can't we create new consts which aren't a member of devnetConsensus that represents the default values?

Comment (UdjinM6) [reply src/init.cpp:620]: How about https://github.com/UdjinM6/dash/commit/00f109eaa54e18d165d0d37410235b995b195551 ?

Comment (PastaPastaPasta) [reply src/init.cpp:620]: I think that's better...

**Review by xdustinface — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Review by thephez — APPROVED** [review]
utACK

**Pull request merged**